my-app/src/App/pages/MarketPage/components/CategoryCoin/CategoryCoin.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import styles from "./CategoryCoin.module.scss";
+
+const CategoryCoin = () => {
+  // находиться в разработке, поэтому нет useState
+  // const [active, setActive] = useState();
+  const categories = [
+    { id: 1, content: "All", isActive: true },
+    { id: 2, content: "Gainer", isActive: false },
+    { id: 3, content: "Loser", isActive: false },
+    { id: 4, content: "Favourites", isActive: false },
+  ];
+  const handleClickMenu = (e: React.MouseEvent) => {
+    categories.map((category) => (category.isActive = false));
+  };
+  return (
+    <div className={styles.category}>
+      {categories.map((category) => (
+        <h6
+          key={category.id}
+          onClick={handleClickMenu}
+          className={category.isActive ? "active" : ""}
```

Comment:
вариант, как это можно сделать
```suggestion
const CATEGORIES: { id: number; content: string }[] = [
  { id: 1, content: "All" },
  { id: 2, content: "Gainer" },
  { id: 3, content: "Loser" },
  { id: 4, content: "Favourites" },
];

const CategoryCoin = () => {
  const [activeId, setActiveId] = React.useState(1);

  return (
    <div className={styles.category}>
      {CATEGORIES.map((category) => (
        <h6
          key={category.id}
          onClick={() => setActiveId(category.id)}
          className={category.id === activeId ? "active" : ""}
```